### PR TITLE
Fix #316071: Add option to Copy SMuFL Symbol Code for symbols in Master Palette

### PR DESCRIPTION
**Pull request by cbjeukendrup**
Resolves: https://musescore.org/en/node/316071

ezgif-7-6c7cc96908b9

This is an extended version of PR #7301.

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/Documentation/blob/master/CodeGuidelines.md)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: mscore/palette.cpp [hunk 1]
```diff
@@ -274,6 +274,20 @@ void Palette::contextMenuEvent(QContextMenuEvent* event)
...
+                        QString symSmuflName = QString("<sym>%1</sym>").arg(regex.match(cell->name).captured(1));
+                        QApplication::clipboard()->setText(symSmuflName);
+                        }
+                  }
+            return;
+            }
+
       QMenu menu;
       QAction* deleteCellAction   = menu.addAction(tr("Delete"));
       QAction* contextAction = menu.addAction(tr("Properties…"));
```
Comment (Jojo-Schmitz) [inline mscore/palette.cpp:293]: Can't we get rid of these two options in master symbols palette, where neither does anything anyway?

File: mscore/symboldialog.cpp [hunk 2]
```diff
@@ -57,7 +58,7 @@ void SymbolDialog::createSymbols()
                || Sym::id2userName(id).contains(search->text(), Qt::CaseInsensitive)) {
                   Symbol* s = new Symbol(gscore);
                   s->setSym(SymId(id), f);
-                  sp->append(s, Sym::id2userName(SymId(id)));
+                  sp->append(s, qApp->translate("symUserNames", Sym::id2userName(SymId(id)).toUtf8()) + " \"<sym>" + Sym::id2name(SymId(id)) + "</sym>\"");
```
Comment (Jojo-Schmitz) [inline mscore/symboldialog.cpp:61]: Shamelessly stolen from #7301  ;-)

**Review by Jojo-Schmitz — APPROVED** [review]

File: mscore/palette.cpp (hunk 1 above)
Comment (cbjeukendrup) [reply mscore/palette.cpp:293]: For the symbols section, I removed them with this PR, and for the 'normal' sections, there is somehow no right-click menu at all (right click does for me the same as left click).

Comment (Jojo-Schmitz) [reply mscore/palette.cpp:293]: Ah, right, I missed the `return` statement above

**Pull request merged**